src/ir/properties.h

```diff
@@ -359,26 +358,28 @@ inline bool canEmitSelectWithArms(Expression* ifTrue, Expression* ifFalse) {
 // allocations, though, it doesn't matter what is in "..": there is nothing
 // in the wasm that we can check to find out if the results are the same or
 // not. (In fact, in this case they are always not the same.) So the
-// nondeterminism is "intrinsic."
+// nondeterminism is "intrinsic" and it is because each call to struct.new
```

Comment:
```suggestion
// generativity is intrinsic to the expression and it is because each call to struct.new
```

This sentence could also probably be deleted because "Intrinsic" is no longer part of the property name so does not need to be explained.